Documentation/network/servicemesh/default-deny-ingress-policy.rst

```diff
@@ -0,0 +1,69 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_default_deny_ingress_policy:
+
+***************************
+Default Deny Ingress Policy
+***************************
+
+Let's apply clusterwide network policy to deny all the traffic by default.
```

Comment:
I think it would make sense here to refer to the policy as an object, similar to how we do here: https://docs.cilium.io/en/stable/network/kubernetes/policy/#ciliumclusterwidenetworkpolicy

```suggestion
Let's apply a `CiliumClusterwideNetworkPolicy` to deny all traffic by default:
```